cosmic_ray/operators/provider.py

```diff
@@ -0,0 +1,44 @@
+"""Operator-provider plugin for all core cosmic ray operators.
+"""
+
+import itertools
+
+from . import (binary_operator_replacement,
```

Comment:
I'm developing a preference for explicit absolute imports, so `from cosmic_ray.operators import ...`. I think it's because it makes me think less.

It's only a preference though.